testing/fio-user-params.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
so is this an example of the output from multiplex?

Reply:
I don't know if I actually have multiplex doing this yet, but it will.